components/layout_2020/flexbox/layout.rs

```diff
@@ -1071,6 +1072,11 @@ impl FlexContainer {
             sizes.depends_on_block_constraints,
         )
     }
+
+    #[inline]
+    pub(crate) fn layout_style(&self) -> DefaultLayoutStyle {
+        DefaultLayoutStyle(&self.style)
+    }
```

Comment:
You might be able to avoid this by creating a `From` implementation.

Reply:
I think `LayoutStyle::Default(&self.style)` looks clearer than `(&*self.style).into()`

Reply:
OK, I see what you mean. But in some cases it's needed because the box doesn't have access to its style, so it needs to be passed as an argument. So not sure if the inconsistency is worth it?